$NetBSD: patch-CMakeLists.txt,v 1.2 2024/01/06 19:13:38 wiz Exp $

Make sure library search path is in pkg-config file.
https://github.com/aous72/OpenJPH/pull/116

--- CMakeLists.txt.orig	2024-01-06 19:10:34.138110748 +0000
+++ CMakeLists.txt
@@ -86,7 +86,7 @@ endif()
 
 set(PKG_CONFIG_INCLUDEDIR "\${prefix}/include")
 set(PKG_CONFIG_LIBDIR "\${prefix}/lib")
-set(PKG_CONFIG_LIBS "-lopenjph")
+set(PKG_CONFIG_LIBS "-L\${prefix}/lib -lopenjph")
 
 include(GNUInstallDirs)
 install(TARGETS openjph
